lib/B/Op_private.pm

```diff
@@ -600,6 +600,8 @@ our %defines = (
     OPpARGELEM_HV            =>   4,
     OPpARGELEM_MASK          =>   6,
     OPpARGELEM_SV            =>   0,
+    OPpARG_IF_FALSE          =>  64,
```

Comment:
Is it okay for `OPpARG_IF_FALSE` to be assigned the same value as `OPpASSIGN_BACKWARDS` a few lines below?

Reply:
Yeah this is fine. `OPpARG_IF_FALSE` is for `OP_ARGDEFELEM` whereas `OPpASSIGN_BACKWARDS` is for `OP_SASSIGN`.

The `OPp*` flags are for the `op_private` field, which is only 8 bits wide. The meanings of the bits are "private" to each individual optype. As such there's many many such overlaps and reuses of bits across the different ops. In fact, the entire `regen/op_private` file is used to manage them all and ensure no actual individual op type has a collision.